wkcuber/tile_cubing.py

```diff
@@ -291,6 +289,10 @@ def tile_cubing(
             file_count, tile_size[0], tile_size[1]
         )
     )
+    if args is None or not hasattr(args, "dtype") or args.dtype is None:
+        dtype = image_reader.read_dtype(arbitrary_file)
+        if args is not None:
```

Comment:
If args is None, the dtype isn't passed anywhere if I'm seeing this correctly.

Reply:
Yes, you're correct. :slightly_smiling_face: